docs/content/docs/dev/table/hiveCompatibility/hiveDialect/load-data.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Load Data Statements"
+weight: 4
+type: docs
+aliases:
+- /dev/table/hiveCompatibility/hiveDialect/load.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Load Data Statements
+
+## Description
+
+The `LOAD DATA` statement is used to load the data into a Hive table from the user specified directory or file.
+The load operation are currently pure copy/move operations that move data files into locations corresponding to Hive tables.
```

Comment:
copy/move? It seems it always move the origin file no matter whether it parse the row successfully....

Reply:
There should copy for load local data, move for others. I'll fix the bugs.